### ISSUE TITLE
Feature Idea, docker-hostmaster for docker workflows

### ISSUE THREAD

**Issue by frob** [geerlingguy/drupal-vm#1754]
### Issue Type

 - Feature Idea

### Summary

I recently came across https://github.com/iamluc/docker-hostmanager and thought about how nice it would be for us to have proper hostmanager support in docker based workflows.

**Comment by fubarhouse** [comment]
@frob,

I really like it, but it hasn't been updated in 2 years.

Would you know the maintenance status of the project?

**Comment by frob** [comment]
It looks like it is being actively (at the moment) maintained. While nothing has been committed in 2 years the maintainer was commenting on PRs as of 5 days ago. Sometimes things just work and don't need new releases. 

That said, I haven't used this project, I am just getting started in adopting Docker for development and as such I am fully unqualified to review if project support should be included. I am just looking into all the things that I will need for docker to be a viable replacement for vagrant.

**Comment by frob** [comment]
Makes sense, for what it is worth. I wasn't able to get this to work at all. So closed is likely the best place for it.
